privval/socket_listeners_test.go

```diff
@@ -122,5 +129,9 @@ func TestListenerTimeoutReadWrite(t *testing.T) {
 		if have, want := opErr.Op, "read"; have != want {
 			t.Errorf("for %s listener, have %v, want %v", tc.description, have, want)
 		}
+
+		if have, want := opErr.Timeout(), true; have != want {
```

Comment:
```suggestion
		if !opErr.Timeout() {
```

Reply:
I like to have this consistent with the cases above (would you also change to `if opErr.Op != "read"`)

Reply:
Yes, I would

Reply:
The reason for this consistent form is that when you reformulate the expectations, you can't forget to change the error reprted. Less error prone and less editing.

Reply:
`for %s listener, got unexpected error: have false want true`
in it's current form does not make sense for person running tests

Reply:
@xla agree if you're talking about above case where we compare Op == read, but not here

Reply:
Agreed, it doesn't always produces the most readable output. A balance must be struck.